### DISCUSSION TITLE
Only some meeting populate when looking at all days

### DISCUSSION THREAD

**Discussion by MarWood4** [code4recovery/12-step-meeting-list#799] [Q&A] [answered]
On our site, we're limited to 20 items that feed for some reason with your plug-in creator.
How do I increase that. See this page for example. I have 33 items i want to display, but only 20 appear. https://aadistrict15mi.com/meetings/

**Comment by joshreisner** [comment, marked as answer]
huh, sorry about that! TSML UI renders meetings in batches of 10 as you scroll down the page so as not to overload the browser when there are a lot of meetings.

but for some reason, on your site, the scroll event is not firing properly when you reach the bottom. 

are you able to add custom CSS on your site? if so, try adding this:

```
body.tsml-ui {
  height: auto;
  min-height: 100%;
}
```

this should only affect the pages that TSML UI is on and hopefully allow the infinite scroller to work properly

**Comment by MarWood4** [comment]
Worked, thanks!On May 3, 2022, at 9:52 AM, Josh Reisner ***@***.***> wrote:huh, sorry about that! TSML UI renders meetings in batches of 10 as you scroll down the page so as not to overload the browser when there are a lot of meetings.but for some reason, on your site, the scroll event is not firing properly when you reach the bottom.are you able to add custom CSS on your site? if so, try adding this:body.tsml-ui {
  height: auto;
  min-height: 100%;
}
this should only affect the pages that TSML UI is on and hopefully allow the infinite scroller to work properly—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
